integration-tests/features/gemini.feature

```diff
@@ -36,8 +36,6 @@ Feature: Gemini Analytics API
   Scenario: Check the Gemini API /api/v1/user-repo/scan response
     Given Gemini service is running
         And Gemini service git url is https://github.com/jitpack/maven-simple
```

Comment:
whats the significance of this repo here? will it get over ridden by the node repo provided later?
cc: @tisnik

Reply:
@yzainee better to ask Agam or @samuzzal-choudhury as they are authors.

Reply:
We need to change this feature file accordingly, if we changed the endpoint behavior.